SparkPost/2web2ui
# [AC-1836] UI: Enforce 2FA required changes

[AC-1836] UI: Enforce 2FA required changes

### What Changed

- When account option `enforce_tfa_required` is enabled then 
     - On account settings page, “2FA Required” status should read “Required” and the toggle button should be disabled at 
     “On” position 

     - Show a banner under the 2FA panel saying: “Two-factor authentication is required for your account.” 

     - The SSO section should be hidden when the when the account option “enforce_tfa_required” is true.


- When tfa_required is true then Disable 2FA should read Reset 2FA

### How To Test

- Enable ''enforce_tfa_required" for your account and verify the state mentioned in 1st point.
- without "enforce_tfa_required" banner shouldn't be visible  and SSO section shouldn't be hidden.
- Enable "tfa_required" and verify the state mentioned in 2nd point.

### To Do



[AC-1836]: https://sparkpost.atlassian.net/browse/AC-1836
